ReSharper.FSharp/src/FSharp/FSharp.Common/src/Settings/FSharpOptions.fs

```diff
@@ -173,13 +177,23 @@ module FSharpTypeHintOptions =
     let [<Literal>] hideSameLinePipe = "Hide when |> is on same line as argument"
 
 
-[<SettingsKey(typeof<FSharpOptions>, "FSharpTypeHintOptions")>]
+[<SettingsKey(typeof<FSharpOptions>, nameof(FSharpTypeHintOptions))>]
 type FSharpTypeHintOptions =
     { [<SettingsEntry(true, FSharpTypeHintOptions.pipeReturnTypes); DefaultValue>]
       mutable ShowPipeReturnTypes: bool
 
       [<SettingsEntry(true, FSharpTypeHintOptions.hideSameLinePipe); DefaultValue>]
-      mutable HideSameLine: bool }
+      mutable HideSameLine: bool
+
+      [<SettingsEntry(PushToHintMode.Default,
+                      DescriptionResourceType = typeof<Strings>,
+                      DescriptionResourceName = nameof(Strings.FSharpTypeHints_TopLevelMembers_Description))>]
+      mutable ShowTypeHintsForTopLevelMembers: PushToHintMode
+
+      [<SettingsEntry(PushToHintMode.PushToShowHints,
+                      DescriptionResourceType = typeof<Strings>,
+                      DescriptionResourceName = nameof(Strings.FSharpTypeHints_LocalBindings_Description))>]
+      mutable ShowTypeHintsForLocalBindings: PushToHintMode }
```

Comment:
Perhaps we can make a different default here, but this one seemed most appropriate to me